### PR TITLE
Eliminate GHC/Util/Module

### PR DESCRIPTION
**Pull request by shayne-fletcher**
Tiny:
 -  Eliminate `GHC.Util.Module`; 
 - Update `ghc-lib-parser-ex` (for `modName` and pin `extra-1.7.3`).

## 8.10.0.12 released 2020-05-31
- New module `GHC.Hs`

## 8.10.0.13 released 2020-05-31
- Sync `extra` with HLint

### CODE REVIEW

**Comment by shayne-fletcher** [comment]
Travis seems to be ignoring me on this one 😀

File: stack.yaml [hunk 1]
```diff
@@ -3,11 +3,11 @@ packages:
...
 extra-deps:
   - ghc-lib-parser-8.10.1.20200523
-  - ghc-lib-parser-ex-8.10.0.11
+  - ghc-lib-parser-ex-8.10.0.13
 # To test hlint against experimental builds of ghc-lib-parser-ex,
 # modify extra-deps like this:
-#  - archive: /users/shaynefletcher/project/ghc-lib-parser-ex.git/ghc-lib-parser-ex-8.10.0.11.tar.gz
-#    sha256: "0000000000000000000000000000000000000000000000000000000000000000"
+#  - archive: /users/shaynefletcher/project/ghc-lib-parser-ex.git/ghc-lib-parser-ex-8.10.0.14.tar.gz
+#    sha256: "0000000000000000000000000000000000000000000000000000000000000000
```
Comment (ndmitchell) [inline stack.yaml:10]: You lost the trailing `"` again...

File: stack.yaml [hunk 2]
```diff
@@ -3,11 +3,11 @@ packages:
   - .
 extra-deps:
   - ghc-lib-parser-8.10.1.20200523
-  - ghc-lib-parser-ex-8.10.0.11
+  - ghc-lib-parser-ex-8.10.0.13
 # To test hlint against experimental builds of ghc-lib-parser-ex,
 # modify extra-deps like this:
-#  - archive: /users/shaynefletcher/project/ghc-lib-parser-ex.git/ghc-lib-parser-ex-8.10.0.11.tar.gz
-#    sha256: "0000000000000000000000000000000000000000000000000000000000000000"
+#  - archive: /users/shaynefletcher/project/ghc-lib-parser-ex.git/ghc-lib-parser-ex-8.10.0.14.tar.gz
```
Comment (ndmitchell) [inline stack.yaml:9]: The 14 and the 13 above aren't matched.

**Comment by shayne-fletcher** [comment]
I'll fix it up. Thanks for pointing these things out. Stand by.

File: stack.yaml (hunk 2 above)
Comment (shayne-fletcher) [reply stack.yaml:9]: That's ok, that should read 14 not that it's important.

File: stack.yaml (hunk 1 above)
Comment (shayne-fletcher) [reply stack.yaml:10]: Fixed.

**Comment by shayne-fletcher** [comment]
Done. Ha, Travis is back.

**Comment by ndmitchell** [comment]
Thanks

**Pull request merged**